lib/companies/job_checker.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Companies.JobChecker do
+  alias Companies.Jobs
+
+def check_job(%{url: url} = job) do
+  case HTTPoison.head(url) do
+    {:ok, %{status_code: 404}} -> Jobs.delete(job)
+    _ -> nil
+  end
+end
+    case job.url |> HTTPoison.head() do
```

Comment:
I think something happened here.

Reply:
yup, github happened with their "apply changes" feature: https://github.com/doomspork/elixir-companies/pull/483/commits/ea2efe86411d8cae1ec49eb423b8c4039ccf1d14

Reply:
There is still an issue here, can you change it and push it again to check? thnx :)